src/site/stages/build/process-cms-exports/schemas/page.js

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable camelcase */
+const { getFilter } = require('../filters');
+
+module.exports = {
+  type: 'object',
+  properties: {
```

Comment:
Should we also have `metatag` in here now?

Reply:
Probably. I need to download the one with metatags...

Reply:
[Done](https://github.com/department-of-veterans-affairs/vets-website/pull/10982/commits/eaccbba69ed7107020544a4ab601d970b43baeb8)!